### PR TITLE
New output dens

### PR DESCRIPTION
**Pull request by rtei-pw**

### CODE REVIEW

File: src/solver/cmake/variable.cmake [hunk 1]
```diff
@@ -129,6 +129,7 @@ set(SRC_VARIABLE_ECONOMY
 		variable/economy/waterValue.h
 		variable/economy/hydroCost.h
 		variable/economy/unsupliedEnergy.h
+		variable/economy/domesticUnsupliedEnergy.h
```
Comment (hugo-antoine-rtei) [inline src/solver/cmake/variable.cmake]: switch domesticUnsupliedEnergy for domesticUnsuppliedEnergy, and the same for the other places

**Review by hugo-antoine-rtei — CHANGES_REQUESTED** [review]

File: src/solver/cmake/variable.cmake (hunk 1 above)
Comment (rtei-pw) [reply src/solver/cmake/variable.cmake]: ok

**Comment by hugo-antoine-rtei** [comment]
Thank you @rtei-pw, I don't know why RTE kept the spelling mistake

**Review by hugo-antoine-rtei — APPROVED** [review]

**Comment by rtei-pw** [comment]
@Milos-RTEi can you check if anything needs to be changed as described in the new spec:
Outputs: 

The DENS will be added in the outputs of Antares, under the name **LOCAL UNSP. ENRG**, for all selected areas. This output will be available in the General values tab, as year-by-year and synthetic results. **All the statistic values for it are to be calculated as well (min, max, mean, standard deviation)**. The output needs to be stored as the other values, in the values-**annual** & values-**hourly** output files. For the areas not selected, the output **LOCAL UNSP. ENRG will be 0.**

**Comment by Milos-RTEi** [comment]
> @Milos-RTEi can you check if anything needs to be changed as described in the new spec: Outputs:
> 
> The DENS will be added in the outputs of Antares, under the name **LOCAL UNSP. ENRG**, for all selected areas. This output will be available in the General values tab, as year-by-year and synthetic results. **All the statistic values for it are to be calculated as well (min, max, mean, standard deviation)**. The output needs to be stored as the other values, in the values-**annual** & values-**hourly** output files. For the areas not selected, the output **LOCAL UNSP. ENRG will be 0.**

Changed the name of the column and tested all the characteristic from the comment/specifications. It's OK.

File: src/solver/variable/economy/domesticUnsuppliedEnergy.h [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    {
+        // Next variable
+        NextType::hourBegin(hourInTheYear);
+    }
+
+    void hourForEachArea(State& state, unsigned int numSpace)
+    {
+        // Total DomesticUnsuppliedEnergy emissions
+        pValuesForTheCurrentYear[numSpace][state.hourInTheYear]
+          = 123; // Connect to DENS values in the similar manner like below
```
Comment (rtei-pw) [inline src/solver/variable/economy/domesticUnsuppliedEnergy.h]: change it to 0

Comment (Milos-RTEi) [reply src/solver/variable/economy/domesticUnsuppliedEnergy.h]: Sure. Now when I'm done testing the solution I'll put 0. It impossible to test with the zero.

**Pull request merged**